relay_api.go

```diff
@@ -44,7 +44,7 @@ type RelayCall interface {
 	Destination() (peer *Peer, ok bool)
 
 	// SentBytes is called when a frame is sent to the destination peer.
-	SentBytes(uint16)
+	SentBytes(uint)
```

Comment:
it's a little odd to have `uint` here, but `uint16` for received bytes. Some alternatives:
 * call `SentBytes` for each frame, so we can keep it `uint16`
 * modify both `SentBytes` and `ReceivedBytes` to `uint`

The first has no impact on Muttley. The latter will need some minor fixes in Muttley since it's technically a breaking change (but since this is an internal API, that's not a big deal)

Reply:
went with the second approach